contracts/v3/VaultHelper.sol

```diff
@@ -62,7 +62,6 @@ contract VaultHelper {
     )
         external
     {
-        require (_amount > 0);
```

Comment:
Just removing alone will not fix this. If we want to have consistent checks, then I would say that we should add a check to the for loop:

```solidity

        for (uint8 i = 0; i < _amounts.length; i++) {
            uint amount = _amounts[i];
            if (amount > 0) {
                IERC20(_tokens[i]).safeTransferFrom(msg.sender, address(this), _amounts[i]);
                IERC20(_tokens[i]).safeApprove(_vault, 0);
                IERC20(_tokens[i]).safeApprove(_vault, _amounts[i]);
            }
        }
```

Reply:
Could also do:
```JS
if (amount != 0) {   // slightly cheaper
```
If amount ==0 should never happen:
```JS
require(amount != 0,"Zero");       // e.g. in the loop, instead of "if (amount > 0) {"
```